src/main/kotlin/demo.kt

```diff
@@ -43,10 +49,11 @@ fun main() {
                 }
             }
             ul {
-                seq.each().map { s: String ->
+                seq.each().mapIndexed{ i:Int, s: String ->
```

Comment:
new method or the map-method get a second parameter with index!?

Reply:
definitely a second method. different semantics. but I still do not think, it is a good idea.